<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scan_book</class>
 <widget class="QWidget" name="scan_book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>scan_book</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Drag &amp; Drop PDF file here</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_pageBoards"/>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_previous">
           <property name="text">
            <string>Previous</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_page">
           <property name="text">
            <string>Page</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_next">
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="BoardView" name="board_view" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BoardView</class>
   <extends>QWidget</extends>
   <header>GUI/components/BoardView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_previous</sender>
   <signal>clicked()</signal>
   <receiver>scan_book</receiver>
   <slot>on_previous_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_next</sender>
   <signal>clicked()</signal>
   <receiver>scan_book</receiver>
   <slot>on_next_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>-22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget_pageBoards</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>scan_book</receiver>
   <slot>on_board_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>-13</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_previous_clicked()</slot>
  <slot>on_next_clicked()</slot>
  <slot>on_board_changed()</slot>
 </slots>
</ui>
